### PR TITLE
sysvar: add 12 missing planner sysvars

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
sysvar: add 12 missing planner sysvars

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/9943
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1673,6 +1787,24 @@ explain select * from t where age=5;
...
 - 指定是否允许优化器将 `Projection` 算子下推到 TiKV 或者 TiFlash。
 
+### `tidb_opt_scan_factor`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：浮点数
+- 范围：`[0, 2147483647]`
+- 默认值：`1.5`
+- 表示在升序排序中，TiKV 在磁盘上扫描一行数据的开销。该变量是[代价模型](/cost-model.md)内部使用的变量，不建议修改该变量的值。
```
Comment (qw4990) [inline system-variables.md:1797]: `升序排序` --> `升序扫描`

File: system-variables.md [hunk 2]
```diff
@@ -1544,6 +1613,33 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
+- 表示 TiDB 处理一行数据的 CPU 开销。该变量是[代价模型](/cost-model.md)内部使用的变量，不建议修改该变量的值。
+
+### `tidb_opt_desc_scan_factor`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：浮点数
+- 范围：`[0, 2147483647]`
+- 默认值：`3.0`
+- 表示在降序排序中，TiKV 在磁盘上扫描一行数据的开销。该变量是[代价模型](/cost-model.md)内部使用的变量，不建议修改该变量的值。
```
Comment (qw4990) [inline system-variables.md:1632]: `降序排序` --> `降序扫描`

Comment (TomShawn) [reply system-variables.md:1632]:
```suggestion
- 表示在降序扫描中，TiKV 在磁盘上扫描一行数据的开销。该变量是[代价模型](/cost-model.md)内部使用的变量，不建议修改该变量的值。
```

File: system-variables.md (hunk 1 above)
Comment (TomShawn) [reply system-variables.md:1797]:
```suggestion
- 表示升序扫描时，TiKV 在磁盘上扫描一行数据的开销。该变量是[代价模型](/cost-model.md)内部使用的变量，不建议修改该变量的值。
```

**Review by qw4990 — APPROVED** [review]
LGTM

File: system-variables.md [hunk 3]
```diff
@@ -1544,6 +1613,33 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
+- 表示 TiDB 处理一行数据的 CPU 开销。该变量是[代价模型](/cost-model.md)内部使用的变量，不建议修改该变量的值。
+
+### `tidb_opt_desc_scan_factor`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：浮点数
+- 范围：`[0, 2147483647]`
+- 默认值：`3.0`
+- 表示降序扫描时，TiKV 在磁盘上扫描一行数据的开销。该变量是[代价模型](/cost-model.md)内部使用的变量，不建议修改该变量的值。
```
Comment (Oreoxmt) [inline system-variables.md:1632]:
```suggestion
- 表示降序扫描时，TiKV 在磁盘上扫描一行数据的开销。该变量是[代价模型](/cost-model.md)内部使用的变量，**不建议**修改该变量的值。
```

File: system-variables.md [hunk 4]
```diff
@@ -1544,6 +1613,33 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
+
+### `tidb_opt_disk_factor`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：浮点数
+- 范围：`[0, 2147483647]`
+- 默认值：`1.5`
+- 表示 TiDB 往临时磁盘读写一个字节数据的 I/O 开销。该变量是[代价模型](/cost-model.md)内部使用的变量，不建议修改该变量的值。
```
Comment (Oreoxmt) [inline system-variables.md:1641]:
```suggestion
- 表示 TiDB 往临时磁盘读写一个字节数据的 I/O 开销。该变量是[代价模型](/cost-model.md)内部使用的变量，**不建议**修改该变量的值。
```

File: system-variables.md [hunk 5]
```diff
@@ -1625,6 +1721,15 @@ mysql> desc select count(distinct a) from test.t;
...
 - 如果 Limit 或者 TopN 的取值小于等于这个阈值，则 Limit 和 TopN 算子会被强制下推到 TiKV。该变量可以解决部分由于估算误差导致 Limit 或者 TopN 无法被下推的问题。
 
+### `tidb_opt_memory_factor`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：浮点数
+- 范围：`[0, 2147483647]`
+- 默认值：`0.001`
+- 表示 TiDB 存储一行数据的内存开销。该变量是[代价模型](/cost-model.md)内部使用的变量，不建议修改该变量的值。
```
Comment (Oreoxmt) [inline system-variables.md:1731]:
```suggestion
- 表示 TiDB 存储一行数据的内存开销。该变量是[代价模型](/cost-model.md)内部使用的变量，**不建议**修改该变量的值。
```

File: system-variables.md [hunk 6]
```diff
@@ -1633,6 +1738,15 @@ mysql> desc select count(distinct a) from test.t;
...
 - 当该变量值为 `ON` 时，左连接始终使用内表作为构建端，右连接始终使用外表作为构建端。将该变量值设为 `OFF` 后，外连接可以灵活选择任意一边表作为构建端。
 
+### `tidb_opt_network_factor`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化
+- 类型：浮点数
+- 范围：`[0, 2147483647]`
+- 默认值：`1.0`
+- 表示传输 1 比特数据的网络净开销。该变量是[代价模型](/cost-model.md)内部使用的变量，不建议修改该变量的值。
```
Comment (Oreoxmt) [inline system-variables.md:1748]:
```suggestion
- 表示传输 1 比特数据的网络净开销。该变量是[代价模型](/cost-model.md)内部使用的变量，**不建议**修改该变量的值。
```

File: system-variables.md [hunk 7]
```diff
@@ -1673,6 +1787,24 @@ explain select * from t where age=5;
...
 - 指定是否允许优化器将 `Projection` 算子下推到 TiKV 或者 TiFlash。
 
+### `tidb_opt_scan_factor`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：浮点数
+- 范围：`[0, 2147483647]`
+- 默认值：`1.5`
+- 表示升序扫描时，TiKV 在磁盘上扫描一行数据的开销。该变量是[代价模型](/cost-model.md)内部使用的变量，不建议修改该变量的值。
```
Comment (Oreoxmt) [inline system-variables.md:1797]:
```suggestion
- 表示升序扫描时，TiKV 在磁盘上扫描一行数据的开销。该变量是[代价模型](/cost-model.md)内部使用的变量，**不建议**修改该变量的值。
```

File: system-variables.md [hunk 8]
```diff
@@ -1673,6 +1787,24 @@ explain select * from t where age=5;
...
+
+### `tidb_opt_seek_factor`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：浮点数
+- 范围：`[0, 2147483647]`
+- 默认值：`20`
+- 表示 TiDB 从 TiKV 请求数据的初始开销。该变量是[代价模型](/cost-model.md)内部使用的变量，不建议修改该变量的值。
```
Comment (Oreoxmt) [inline system-variables.md:1806]:
```suggestion
- 表示 TiDB 从 TiKV 请求数据的初始开销。该变量是[代价模型](/cost-model.md)内部使用的变量，**不建议**修改该变量的值。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**